scripts/inner/kubestellar

```diff
@@ -88,10 +88,10 @@ function ensure_imws() {
             imw=${i#"root:"}
             echo "Ensuring IMW: $imw"
             space_config="${KUBECONFIG_DIR}/${PROVIDER_NAMESPACE}-${imw}"
```

Comment:
`$PROVIDER_NAMESPACE` is not a variable that varies, right? Can we just remove its appearance here?